resources/views/tasks.blade.php

```diff
@@ -4,12 +4,17 @@
         <div class="card">
             <h2 class="card-header">{{ trans('messages.new_task_title') }}</h2>
             @include('common.errors')
+            @if ( $task_message = session('task_message') )
```

Comment:
```suggestion
            @if ( $taskMessage = session('task_message') )
```
sửa lại những phần tương tự

Reply:
Em đã sửa.